install/Knative-with-ICP.md

```diff
@@ -0,0 +1,170 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. See [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html) in the IBM Knowledge Center for install instructions.
+
+1: [Install Docker for your boot node only](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#prep_boot)
+
+2: [Set up the installation environment](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#setup)
+
+3: [Customize your cluster](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#customize)
+
+4: [Set up Docker for your cluster nodes](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#cluster_nodes)
+
+5: [Deploy the environment](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#deploy)
+
+6: [Verify the status of your installation](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html#verify)
+
+## Installing Istio
+
+Follow the [instructions](https://istio.io/docs/setup/kubernetes/quick-start-ibm/#ibm-cloud-private) to install and run Istio in [IBM Cloud Private](https://www.ibm.com/cloud/private).
+
+## Installing Knative Serving
+
+Next, install [Knative Serving](https://github.com/knative/serving):
```

Comment:
Can you number these steps or add additional formatting so it's easier to follow?